op-e2e/bridge_test.go

```diff
@@ -119,3 +125,131 @@ func TestERC20BridgeDeposits(t *testing.T) {
+	l2Opts, err := bind.NewKeyedTransactorWithChainID(sys.Cfg.Secrets.Bob, cfg.L2ChainIDBig())
+	require.Nil(t, err)
+
+	// Init bridge contracts
+	l1Bridge, err := bindings.NewL1StandardBridge(l1BridgeAddr, l1Client)
+	require.NoError(t, err)
+	l2Bridge, err := bindings.NewL2StandardBridge(l2BridgeAddr, l2Client)
+	require.NoError(t, err)
+
+	l1Token, err := bindings.NewGovernanceToken(l1TokenAddr, l1Client)
+	require.NoError(t, err)
+	l2Token, err := bindings.NewGovernanceToken(l2TokenAddr, l2Client)
+	require.NoError(t, err)
+
+	// Approve GovernanceToken with the bridge on L1 and L2
+	tx, err := l1Token.Approve(l1Opts, l1BridgeAddr, new(big.Int).SetUint64(math.MaxUint64))
+	require.NoError(t, err)
+	_, err = wait.ForReceiptOK(context.Background(), l1Client, tx.Hash())
+	require.NoError(t, err)
+	tx, err = l2Token.Approve(l2Opts, l2BridgeAddr, new(big.Int).SetUint64(math.MaxUint64))
+	require.NoError(t, err)
+	_, err = wait.ForReceiptOK(context.Background(), l2Client, tx.Hash())
+	require.NoError(t, err)
+
+	// Wait until Alice have enough tokens on L2
+	_, err = geth.WaitForBlock(big.NewInt(10), l2Client, 20*time.Second)
+	require.NoError(t, err)
+	aliceL2Opts, err := bind.NewKeyedTransactorWithChainID(cfg.Secrets.Alice, cfg.L2ChainIDBig())
+	require.Nil(t, err)
+	bridgeAmount, err := l2Token.BalanceOf(&bind.CallOpts{}, aliceL2Opts.From)
+	require.NoError(t, err)
+	require.NotZero(t, bridgeAmount.Uint64())
+
+	// Send Alice's tokens to Bob
+	require.Nil(t, err)
+	tx, err = transactions.PadGasEstimate(aliceL2Opts, 1.1, func(opts *bind.TransactOpts) (*types.Transaction, error) {
+		return l2Token.Transfer(opts, l2Opts.From, bridgeAmount)
+	})
+	require.NoError(t, err)
+	_, err = wait.ForReceiptOK(context.Background(), l2Client, tx.Hash())
+	require.NoError(t, err)
+
+	bobL1Balance, err := l1Token.BalanceOf(&bind.CallOpts{}, l1Opts.From)
+	require.NoError(t, err)
+	require.Zero(t, bobL1Balance.Uint64())
+	bobL2Balance, err := l2Token.BalanceOf(&bind.CallOpts{}, l2Opts.From)
+	require.NoError(t, err)
+	require.Equal(t, bobL2Balance, bridgeAmount)
+
+	// Withdraw GovernanceToken to L1
+	tx, err = transactions.PadGasEstimate(l2Opts, 1.1, func(opts *bind.TransactOpts) (*types.Transaction, error) {
+		return l2Bridge.BridgeERC20(opts, l2TokenAddr, l1TokenAddr, bobL2Balance, 100000, []byte{})
+	})
+	require.NoError(t, err)
+	receipt, err := wait.ForReceiptOK(context.Background(), l2Client, tx.Hash())
+	require.NoError(t, err)
+	version := eth.OutputVersionV0
+	proveReceipt, finalizeReceipt := ProveAndFinalizeWithdrawal(t, version, cfg, l1Client, sys.EthInstances["verifier"], cfg.Secrets.Bob, receipt)
+	require.Equal(t, types.ReceiptStatusSuccessful, proveReceipt.Status)
+	require.Equal(t, types.ReceiptStatusSuccessful, finalizeReceipt.Status)
+
+	// Withdrawal complete, Should have balance on L1
+	bobL1Balance, err = l1Token.BalanceOf(&bind.CallOpts{}, l1Opts.From)
+	require.NoError(t, err)
+	require.NotZero(t, bobL1Balance.Uint64())
```

Comment:
Let's check the balance is zero before withdrawal
And let's assert that the balance after withdrawal is equal to the amount of withdrawal.
Furthermore, it would be good to add assertion that the governance token on L2 has been burnt after withdrawal.